Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3991,12 +3892,8 @@ Optional<CalculatedStyleValue::CalcValue> Parser::parse_calc_value(TokenStream<S
         return (CalculatedStyleValue::CalcValue) { parsed_calc_sum.release_nonnull() };
     }
 
-    if (current_token.is(Token::Type::Number)) {
-        auto try_the_number = try_parse_float(current_token.token().number_string_value());
-        if (try_the_number.has_value())
-            return (CalculatedStyleValue::CalcValue) { try_the_number.value() };
-        return {};
-    }
+    if (current_token.is(Token::Type::Number))
+        return (CalculatedStyleValue::CalcValue) { static_cast<float>(current_token.token().number_value()) };
```

Comment:
Wow, nice C-style struct literal!
(since it was already like this, fixing it would just be a bonus, feel free to not do so)

Reply:
:+1: